codes/classical/properties/dual.yml

```diff
@@ -22,6 +22,9 @@ description: |
 protection: 'The dual of an \([n,k,d] \) code
   is an \([n,n-k,d'']\) code, where \(d''\) is not always related to \(d\). The generator matrix of \(C^\perp\) is the parity check matrix of \(C\), and visa versa.'
 
+notes:
+  - 'See \cite{arXiv:1706.01694}, which contains classing information concerning variations of extremal self-dual codes.'
```

Comment:
I think I edited this sentence last time you submitted; please check older comment. "classing information" is a term that is used.